src/v2/System/Router/buildClientApp.tsx

```diff
@@ -113,7 +113,7 @@ export function buildClientApp(config: RouterConfig): Promise<Resolve> {
             user={user}
             relayEnvironment={relayEnvironment}
             routes={routes}
-            featureFlags={featureFlags}
+            featureFlags={getENV('featureFlags')}
```

Comment:
this is minor and non-blocking, but all of our other ENV variables, even set from sharify, are `FEATURE_FLAGS`, and it would be good to remain consistent here across the board.